README.md

```diff
@@ -1,83 +1,31 @@
-  
-  **Expected outcome:**
-  We should be able to fetch the network printers status as many times as we would like, and not have to worry about the attempt to fetch the status failing. This works as expected on Android - but fails on iOS
-  
-  **Actual outcome:**
-  The first attempt to fetch the printers status will succeed, but any subsequent attempt to grab the printers status will fail with the following error: Failed to getPort. Printer is in use.
-  
---------------------------------------  
-#### Star Bluetooth printer with an active StarIoExtManager connection won't reconnect if we don't call releasePort() after a print job. As a result, calling releasePort() results in the EXC_BAD_ACCESS crashes that we have been seeing in production
-
-  **Steps to reproduce:**
-  1. Manually pair a Star Micronics Bluetooth printer to the iOS device
-  2. On the sample app, click the "Scan BT" button to discover the paired BT printer
-  3. Click on the discovered BT printer row to navigate to printer details
-  4. In the top right, click "Connect" - wait for the connection to succeed 
-  5. When the connection status is "didPrinterOnline, click "Print and do not release port" your printer should then print
-  6. Now, turn the printer off - the status should update to "didPrinterImpossible"
-  7. Turn the printer back on, see that the printer status is now "didAccessoryConnectFailure"
-  8. At this point I would expect it to be "didPrinterOnline"
-
-  **Video:** https://user-images.githubusercontent.com/45129610/144144933-d20680af-ded4-4849-a791-10733853161f.mp4
-
-  **Expected outcome:**
-  The printer should reconnect to the app after a print has completed without releasing the port. We shouldn't need to release the port after a print job, because we are using the StarIoExtManager to secure a connection with the app. 
-  
-  **Actual outcome:**
-  The printer fails the reconnect to the app after a single print job has completed. This means that the user will have to restart their app everytime the printer has been turn off/on.
-
---------------------------------------  
+## iOS Bugs
 
 #### Star Bluetooth printer with an active StarIoExtManager connection won't reconnect after calling disconnect()/connect()
 
+  **Background:**
+  This regression was introduced after addressing another bug which involved swapping between our Point of Sale app and opening another 3rd party app that also makes a connection to the BT printer. Previously we were not calling disconnec() when backgrounding and not calling connect() when foregrounding our app, and would lose connection to the Printer. After adding calls to disconnect/connect when background/foregrounding it addressed our initial connection issue, but introduced a new regression when powering off the BT printer during backgrounding and not being able to connect after powering it back on after foregrounding.
+
```

Comment:
While this is great background information and does describe our bug, I'm wondering if we can be a bit more succinct and direct in giving this info to Star. What do you think about something like this?

```suggestion
  **Bug description:**
This sample app will use a `StarIoExtManager` to connect with a Bluetooth printer.  Once connected, the app will automatically call `starIoExtManager.disconnect()` if the app is subsequently backgrounded, and then `starIoExtManager.connect()` if the app is brought into the foreground again.  However if the printer is switched off while the app is backgrounded, and then switched on _after_ the app has been foregrounded again, the `StarIoExtManager` will fail to reconnect to the printer and report the following error:

<insert error message from Star Micronics SDK>

The only way to get the printer to connect to the sample app again is to force close and restart the app.
```